### ISSUE TITLE
Push July 2019 to Production

### ISSUE THREAD

**Issue by evantrillium** [ODOT-PTS/TNExT#202]
@anniekfifer can you push the July 2019 staging snapshot to production?

Documentation here:
https://sites.google.com/trilliumtransit.com/trillium-operations/software/3rd-parties/tnext?authuser=0#h.p_8E-u6ETjCLk_

**Comment by anniekfifer** [comment]
@evantrillium Sure! is it just step 6 in that doc that needs to be done?

**Comment by evantrillium** [comment]
I believe so - through "Restart the Service"

**Comment by anniekfifer** [comment]
I'll plan to do this tomorrow after 5pm if that's ok

**Comment by Sihackett** [comment]
New issue - July snapshot not ready yet.

**Comment by anniekfifer** [comment]
Waiting on #204

**Comment by evantrillium** [comment]
@anniekfifer I think this is close-able
